---
layout: default
title: Wade R. Roberts
---
{% include section.html %}

I am currently a researcher in the [Alverson Lab](https://alversonlab.com) in the Department of Biological Sciences at the University of Arkansas. I previously completed my Ph.D. in Molecular Plant Sciences with Eric Roalson at Washington State University.

Contact me: wade.r.roberts at gmail (dot) com

{% include button.html label="CV" link="/pdfs/roberts-cv.pdf" %}

{% include button.html label="Resume" link="/pdfs/roberts_resume.pdf" %}

{% include section.html header="Research" %}

I am fascinated by the diversity and complexity of life. My research focuses on the evolutionary processes that give rise to species diversity and uses a combination of phylogenomics and comparative genomics. My primary interest lies in exploring and understanding the interaction between adaptation and genomic complexity. I have used both plant and diatom systems to explore these questions.

For my doctoral work, I studied the transcriptomics of flower diversification in the genus _Achimenes_ (Gesneriaceae), using both phylogenomics and comparative gene expression analyses. My postdoctoral research focuses on the phylogenomics and evolutionary genomics of the centric diatom order Thalassiosirales, an emerging model diatom lineage that spans the marine-freshwater salinity gradient and exhibits substantial genome size variation.

{% include button.html label="Details" link="/research" %}

{% include section.html header="Publications" %}

{% include button.html label="On Google Scholar" link=site.author.scholar icon="fa-graduation-cap" %}

**Roberts W.R.**, Siepielski A.M., Alverson A.J. 2024. Diatom abundance in the polar oceans is predicted by genome size. _PLoS Biology_ [[doi]](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002733)

Bryłka K., Pinseel E, **Roberts W.R.**, Ruck E.C., Conley D.J., Alverson A.J. 2023. Gene duplication, shifting selection, and dosage balance of silicon transporter proteins in marine and freshwater diatoms. _Genome Biology and Evolution_ [[doi]](https://doi.org/10.1093/gbe/evad212)[[pdf]](/pdfs/Brylka_et_al-2023-GBE.pdf?raw=true)

**Roberts W.R.**, Ruck E.C., Downey K.M., Pinseel E., Alverson A.J. 2023. Resolving marine-freshwater transitions by diatoms through a fog of gene tree discordance. _Systematic Biology_ [[doi]](https://doi.org/10.1093/sysbio/syad038)[[pdf]](/pdfs/Roberts_et_al-2023-Systematic_Biology.pdf?raw=true)

Çiftçi O., Alverson A.J., van Bodegom P., **Roberts W.R.**, Mertens A., van de Vijver B., Trobajo R., Mann D., Pirovano W., van Eijk I., Gravendeel B. 2022. Phylotranscriptomics reveals the reticulate evolutionary history of a widespread diatom species complex. _Journal of Phycology_ [[doi]](https://doi.org/10.1111/jpy.13281)[[pdf]](/pdfs/Ciftci_et_al-2022-Journal_of_Phycology.pdf?raw=true)

Onyshchenko A, **Roberts W.R.**, Ruck E.C., Lewis J.A., Alverson A.J. 2021. The genome of a nonphotosynthetic diatom provides insights into the metabolic shift to heterotrophy and constraints on the loss of photosynthesis. _New Phytologist_ [[doi]](https://doi.org/10.1111/nph.17673)[[pdf]](/pdfs/Onyshchenko_et_al-2021-New_Phytologist.pdf?raw=true) [[CoGe genome browser]](https://genomevolution.org/coge/GenomeInfo.pl?gid=60130)

Gargas C.B., **Roberts W.R.**, Alverson A.J. 2020. Genome sequences of bacteria associated with the diatom _Cyclotella cryptica_ strain CCMP332. _Microbiology Resource Announcements_ [[doi]](https://doi.org/10.1128/MRA.01030-20)[[pdf]](/pdfs/Gargas_et_al-2020-MRA.pdf?raw=true)

**Roberts W.R.**, Downey K.M., Ruck E.C., Traller J.C., Alverson A.J. 2020. Improved reference genome for _Cyclotella cryptica_ CCMP332, a model for cell wall morphogenesis, salinity adaptation, and lipid production in diatoms (Bacillariophyta). _G3: Genes, Genomes, Genetics_ [[doi]](https://doi.org/10.1534/g3.120.401408)[[pdf]](/pdfs/Roberts_et_al-2020-G3.pdf?raw=true) [[CoGe genome browser]](https://genomevolution.org/coge/GenomeInfo.pl?gid=57836)

**Roberts W.R.**, Roalson E.H. 2020. Co-expression clustering across flower development identifies modules for diverse floral forms in _Achimenes_ (Gesneriaceaea). _PeerJ_  [[doi]](https://doi.org/10.7717/peerj.8778)[[pdf]](/pdfs/Roberts_et_al_2020-PeerJ.pdf?raw=true)

Kleinkopf J.A., **Roberts W.R.**, Wagner W.L, Roalson E.H. 2019. Diversification of Hawaiian _Cyrtandra_ (Gesneriaceae) under the influence of incomplete lineage sorting and hybridization. _Journal of Systematics and Evolution_ [[doi]](https://doi.org/10.1111/jse.12519)[[pdf]](/pdfs/Kleinkopf_et_al-2019-Journal_of_Systematics_and_Evolution.pdf?raw=true)

**Roberts W.R.**, Roalson E.H. 2018. Phylogenomic analyses reveal extensive gene flow within the magic flowers (_Achimenes_). _American Journal of Botany_ [[doi]](https://doi.org/10.1002/ajb2.1058) [[cover image]](https://onlinelibrary.wiley.com/doi/abs/10.1002/ajb2.1088)[[pdf]](/pdfs/Roberts_et_al-2017-American_Journal_of_Botany.pdf?raw=true)

**Roberts W.R.**, Roalson E.H. 2017. Comparative transcriptome analyses of flower development in four species of _Achimenes_ (Gesneriaceae). _BMC Genomics_ [[doi]](https://doi.org/10.1186/s12864-017-3623-8) [[pdf]](/pdfs/Roberts_et_al_2017-BMC_Genomics.pdf?raw=true)

Roalson E.H., **Roberts W.R.** 2016. Distinct processes drive diversification in different clades of Gesneriaceae. _Systematic Biology_ [[doi]](https://doi.org/10.1093/sysbio/syw012) [[cover image]](https://academic.oup.com/sysbio/article/65/4/i1/1753138) [[pdf]](/pdfs/Syst%20Biol-2016-Roalson-662-84.pdf?raw=true)

{% include section.html header="Github" %}

{% include button.html label="wrroberts" link=site.author.github icon="fa-github" %}
